regimes/pl/examples/corrective-invoice-pl-pl.yaml

```diff
@@ -0,0 +1,56 @@
+$schema: "https://gobl.org/draft-0/bill/invoice"
+currency: "PLN"
+issue_date: "2023-12-21"
+series: "COR"
+code: "002"
+type: "corrective"
```

Comment:
As far as I understand, this type of invoice should not be supported, it should be `credit-note`.